compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/BasicTextField.kt

```diff
@@ -313,6 +314,8 @@ internal fun BasicTextField(
 
     DisposableEffect(textFieldSelectionState) { onDispose { textFieldSelectionState.dispose() } }
 
+    val overscrollEffect = rememberTextFieldOverscrollEffect()
```

Comment:
Is it a regression after merging? Why should it be in the same PR? Why isn't it AOSP first edit?

Reply:
> Is it a regression after merging?
Yes

> Why should it be in the same PR?
It fixes bugs within the overscroll functionality. If you feel that it should be in another one, please let me know the reasons.

> Why isn't it AOSP first edit?
1. The feature has not beed upstreamed yet.
2. Too long feedback loop.